cli/kubectl_kadalu/__main__.py

```diff
@@ -35,11 +37,51 @@ def version_set_args(name, parser):
     parser.add_parser(name)
 
 
+def check_if_operator_exists():
+    """Check if kadalu operator is pre-existing"""
+
+    is_operator_present = 0 #set to false
+
+    cmd1 = ['sudo', 'kubectl', 'get', 'pods', '-nkadalu']
+    cmd2 = ['grep', '"operator-"']
+    cmd3 = ['grep', '"Running"']
+    cmd4 = ['wc', '-l']
+
+    #debugging..
+    #print(f"Shell style : {' '.join(cmd1)} | {' '.join(cmd2)} | {' '.join(cmd3)} | {' '.join(cmd4)}")
+
+    p1 = Popen(cmd1, stdin=None, stdout=PIPE, stderr=PIPE, universal_newlines=True)
```

Comment:
Do not use all these child processes. As suggested in the comment, run `kubectl get deployments -ojson -nkadalu` and parse the json output as `json.loads(out)`. Look for Deployment name as "operator" (Ref: https://github.com/kadalu/kadalu/blob/devel/templates/operator.yaml.j2#L207)

Reply:
and move this logic inside `install.py` `run` function.